src/Components/Element.php

```diff
@@ -320,15 +329,19 @@ final public function toString(): string
             }
         }
 
+        /** @var string $href */
+        $href = $this->properties['href'] ?? '';
+
         $options = [];
 
         foreach ($this->properties['options'] as $option) {
             $options[] = $option;
         }
 
         return sprintf(
-            '%s<%s;options=%s>%s</>%s',
+            '%s<%s%s;options=%s>%s</>%s',
             str_repeat(' ', (int) ($this->properties['styles']['ml'] ?? 0)),
+            $href === '' ? '' : sprintf('href=%s;', $href),
```

Comment:
not always it will be href, but not sure how should we implement it.. I think its fine

Reply:
The syntax is `<href={$url};{$colors};options={$options}>string</>`: https://github.com/symfony/console/blob/5.3/Tests/Formatter/OutputFormatterTest.php#L266